keyboards/nyquist/keymaps/danielhklein/keymap.c

```diff
@@ -44,48 +53,48 @@ enum custom_keycodes {
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
-/* QWERTY
+/* QWERTY 
  * .----------------------------------------.     .-----------------------------------------.
- * | Esc |   1  |   2  |   3  |   4  |   5  |     |   6  |   7  |   8  |   9  |   0  |  [   |
+ * | Esc |   1  |   2  |   3  |   4  |   5  |     |   6  |   7  |   8  |   9  |   0  |  -   |
  * |-----+------+------+------+------+------|     |------+------+------+------+------+------|
- * | Tab |   Q  |   W  |   E  |   R  |   T  |     |   Y  |   U  |   I  |   O  |   P  |  ]   |
+ * | Tab |   Q  |   W  |   E  |   R  |   T  |     |   Y  |   U  |   I  |   O  |   P  |Enter |
  * |-----+------+------+------+------+------|     |------+------+------+------+------+------|
  * | Caps|   A  |   S  |   D  |   F  |   G  |     |   H  |   J  |   K  |   L  |   ;  |  "   |
  * |-----+------+------+------+------+------|     |------+------+------+------+------+------|
- * |Lshft|   Z  |   X  |   C  |   V  |   B  |     |   N  |   M  |   ,  |   .  |  /   |Enter |
+ * |Lshft|   Z  |   X  |   C  |   V  |   B  |     |   N  |   M  |   ,  |   .  |  /   |Rshft |
  * |-----+------+------+------+------+------|     |------+------+------+------+------+------|
- * |     | Ctrl | LAlt | LGui |  Bspc/Raise |     |   Spc/Lower | Left | Down |  Up  |Right |
+ * |Arrow| LCtrl| LAlt | LGui |  Bspc/Raise |     |   Spc/Lower | RGui | RAlt | RCtrl| Del  |
  * `----------------------------------------'     '-----------------------------------------'
  */
 
-[_QWERTY] = LAYOUT(
-  KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,      KC_9,    KC_0,    KC_LBRACKET,  \
-  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,      KC_O,    KC_P,    KC_RBRACKET, \
+[_QWERTY] = KEYMAP(
```

Comment:
Could you change the `KEYMAP`s back to `LAYOUT`s? 

We're depreciating KEYMAP in favor of LAYOUT (which is probably why these got changed).

Reply:
I changed it back to `KEYMAP` because `LAYOUT` wasn't building properly for me. Happy to use `LAYOUT` if I can figure out the cause of these `make` errors:

```
Compiling: keyboards/nyquist/keymaps/danielhklein/keymap.c                                         keyboards/nyquist/keymaps/danielhklein/keymap.c:70:1: error: implicit declaration of function ‘LAYOUT’ [-Werror=implicit-function-declaration]
 [_QWERTY] = LAYOUT(
 ^
keyboards/nyquist/keymaps/danielhklein/keymap.c:76:3: error: missing braces around initializer [-Werror=missing-braces]
   ),
   ^
keyboards/nyquist/keymaps/danielhklein/keymap.c:76:3: error: (near initialization for ‘keymaps[0]’) [-Werror=missing-braces]
keyboards/nyquist/keymaps/danielhklein/keymap.c:76:3: error: initializer element is not constant
keyboards/nyquist/keymaps/danielhklein/keymap.c:76:3: error: (near initialization for ‘keymaps[0][0][0]’)
keyboards/nyquist/keymaps/danielhklein/keymap.c:98:1: error: initializer element is not constant
 ),
 ^
keyboards/nyquist/keymaps/danielhklein/keymap.c:98:1: error: (near initialization for ‘keymaps[2][0][0]’)
keyboards/nyquist/keymaps/danielhklein/keymap.c:120:3: error: initializer element is not constant
   ),
   ^
keyboards/nyquist/keymaps/danielhklein/keymap.c:120:3: error: (near initialization for ‘keymaps[1][0][0]’)
keyboards/nyquist/keymaps/danielhklein/keymap.c:142:1: error: initializer element is not constant
 ),
 ^
keyboards/nyquist/keymaps/danielhklein/keymap.c:142:1: error: (near initialization for ‘keymaps[3][0][0]’)
cc1: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
tmk_core/rules.mk:357: recipe for target '.build/obj_nyquist_rev1_danielhklein/keyboards/nyquist/keymaps/danielhklein/keymap.o' failed
make[1]: *** [.build/obj_nyquist_rev1_danielhklein/keyboards/nyquist/keymaps/danielhklein/keymap.o] Error 1
Makefile:534: recipe for target 'nyquist:danielhklein' failed
Make finished with errors
make: *** [nyquist:danielhklein] Error 1
```

Reply:
Compiles fine on my system.

My guess is that your local repo is out of date.  You may need to rebase or merge the newer changes in.  That should fix the issue.

Reply:
Yep, that did the trick. Fixed.